app/utilities/sortCompanies.js

```diff
@@ -0,0 +1,10 @@
+export function sortCompanies(accordionWrapper) {
+  const elements = Array.from(accordionWrapper.children);
+  const sorted = elements.sort(function (firstElement, secondElement) {
+    return +firstElement.dataset.employees - +secondElement.dataset.employees;
```

Comment:
Don't use implicit type conversion, use explicit conversion instead. Use the `Number()` function to convert something to a number